plutus-contract/plutus-contract.cabal

```diff
@@ -239,10 +239,11 @@ test-suite plutus-contract-test
     Spec.Plutus.Contract.Oracle
     Spec.Plutus.Contract.Wallet
     Spec.Rows
+    Spec.ScriptContext.V1.RequiredSigner
+    Spec.ScriptContext.V1.TimeValidity
```

Comment:
@ak3n @sjoerdvisscher @andreabedini How should we name testing modules in plutus-contract? Let's put ideas here.

Reply:
First off, I would suggest using the package name as the module name and make it consistent:
```
Spec.Plutus.Contract.ScriptContext.V1.RequiredSigner
```

`ScriptContext` might make us think that it is testing on-chain code only, but we're also testing off-chain using the `Contract` monad. Maybe `Spec.Plutus.Contract.IntegrationTest.V1.RequiredSigner` or simply `Spec.Plutus.Contract.V1.RequiredSigner`?

Other than that, I don't have any awesome ideas.

Reply:
I think I like `IntegrationTest` or something similar because these tests are covering a few areas (e.g. Contract, Lookups and ScriptContect's TxInfo)

Reply:
Well, I'd say prepend `Spec` or `Test` to the tested module and use the longest prefix when testing multiple modules but this kinda assumes the module names itself are structured hierarchically.

Reply:
I've moved them into a "Constraints" directory for now, we can decided whether to split by version etc later.